### PR TITLE
sap_install_media_detect: ansible-lint cleanup

### PR DESCRIPTION
**Pull request by berndfinger**

### CODE REVIEW

**Review by sean-freeman — APPROVED** [review]
Tested against multiple scenarios, verified output and actions are identical after these changes

Approved for merge 👍

File: roles/sap_install_media_detect/tasks/cleanup/disable-epel-repo.yml
Comment (rhmk) [inline roles/sap_install_media_detect/tasks/cleanup/disable-epel-repo.yml:1]: Not sure if the first task works. I have the EPEL key installed on my Rhel 8: (even from history RHEL7 key):
```
[mkoch@mkoch ~]$ rpm -q gpg-pubkey --qf '%{NAME}-%{VERSION}-%{RELEASE}\t%{SUMMARY}\n' | grep EPEL
gpg-pubkey-352c64e5-52ae6884	gpg(Fedora EPEL (7) <epel@fedoraproject.org>)
gpg-pubkey-2f86d6a1-5cf7cefb	gpg(Fedora EPEL (8) <epel@fedoraproject.org>)
```
Unfortunately this awk command doesn't find it:
```
rpm -q gpg-pubkey --qf '%{NAME}-%{VERSION}-%{RELEASE}\t%{SUMMARY}\n' | awk '/gpg/&&/Fedora/{gsub ("\\(epel", "EPEL "); gsub (" \\(", " "); if ($0 ~ /EPEL {{ ansible_distribution_major_version }}/)print}'
```

**Review by rhmk — APPROVED** [review]
See my comment regarding the awk command. The Shell commands should not be single- lines (style) because those are very long and hard to read, use e,g, &&\ or \ to include some line breaks.
Nonetheless and due timing, I approve here, but please double check the awk command and split the long shell lines for the next PR.

File: roles/sap_install_media_detect/tasks/cleanup/disable-epel-repo.yml
Comment (berndfinger) [reply roles/sap_install_media_detect/tasks/cleanup/disable-epel-repo.yml:1]: In my test, I am able to find the correct GPG key from the output you showed above:
```echo "gpg-pubkey-352c64e5-52ae6884gpg(Fedora EPEL (7) <epel@fedoraproject.org>)
gpg-pubkey-2f86d6a1-5cf7cefbgpg(Fedora EPEL (8) <epel@fedoraproject.org>)" | awk '/gpg/&&/Fedora/{gsub ("\\(epel", "EPEL "); gsub (" \\(", " "); if ($0 ~ /EPEL 8/)print}'
gpg-pubkey-2f86d6a1-5cf7cefbgpg(Fedora EPEL 8) <epel@fedoraproject.org>)
```
Can you confirm that you tested inside a playbook in which the Ansible variable `ansible_distribution_major_version` is set correctly?

Comment (rhmk) [reply roles/sap_install_media_detect/tasks/cleanup/disable-epel-repo.yml:1]: Ok, missed the ansible vars

Comment (rhmk) [reply roles/sap_install_media_detect/tasks/cleanup/disable-epel-repo.yml:1]: go for it

**Pull request merged**